examples/websocket-relay/relay-app/src/lib/workflow.ts

```diff
@@ -1,16 +1,18 @@
+import { CID } from "multiformats";
+import { base64 } from "iso-base/rfc4648";
```

Comment:
@hugomrdias the imports for `multiformats` and `iso-base` are here.

Reply:
Got that file shows collapsed and seemed to be just removed lines. I will review further. 
Ty

Reply:
Thanks for looking over it! Would be nice to not need `multiformats` here if it's possible to use a string.